### DISCUSSION TITLE
Search/Query span properties

### DISCUSSION THREAD

**Discussion by trhunter92** [getsentry/support#132] [Support] [answered]
Is there a way to query by span properties anywhere in the dashboard? like span.duration, span.tags, etc. I know a lot of these searchable parameters live at the transaction/event level, but I wanted to dig into the spans as well.

I've added tags to some of my spans but still wasn't able to query for them. My use case is that I would like to look for spans over a certain duration within a transaction. Example: I want to find and export all http request spans to Google API that are over 300ms.

Is this something achievable through Sentry?

**Comment by AbhiPrasad** [comment, marked as answer]
Hey, you can accomplish some of this using our [Suspect Spans](https://docs.sentry.io/product/performance/transaction-summary/#suspect-spans) feature. Spans are not indexed in Sentry, so directly querying for them will not work.
